great-idea-website/css/index.css

```
@@ -53,14 +53,92 @@ table {
 }
 
 html, body {
+	box-sizing: border-box;
     height: 100%;
     font-family: 'Titillium Web', sans-serif;
 }
 
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
-    margin-bottom: 15px;
+
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+nav {
+	padding: 30px 0;
+	
+}
+nav a {
+	margin-right: 62px;
+	text-decoration: none;
+	color: #999;
+	display: inline-block;
+}
+h2 {
+	font-size: 20px;
+}
+p {
+	font-size: 18px;
+}
+.container {
+	max-width: 1024px;
+	margin: 0 auto;
+	height: 1536px;
+}
+.contents {
+	height: 400px;
+}
+.top-left {
+	display: inline-block;
+	font-size: 72px;
+	width: 49%;
+	height: 400px;
+}
+.top-right {
+	display: inline-block;
+	font-size: 72px;
+	width: 49%;
+	height: 400px;
+}
+.top-content div {
+	width: 49%;
+}
+.card {
+	display: inline-block;
+	margin-top: 20px;
+}
+.bottom-content div {
+	width: 32%;
+}
+.logo {
+	margin-left: 130px;
+}
+.middle-img {
+	margin-top: 5%;
+	margin-bottom: 2%;
+	width: 100%;
+}
+.contact {
+	margin-top: 5%;
```

```suggestion

	.top-content div {
	width: 49%;
}

.card {
	display: inline-block;
	margin-top: 20px;
}

.bottom-content div {
	width: 32%;
}

.logo {
	margin-left: 130px;
}

.middle-img {
	margin-top: 5%;
	margin-bottom: 2%;
	width: 100%;
}
```  Try and space these out a bit for readability